src/imageio.cs

```diff
@@ -1169,4 +1213,26 @@ interface CGCopyImageSourceOptions {
 		IntPtr kOrientation { get; }
 	}
 #endif
+
+	[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
+	public enum CGImageAuxiliaryDataType {
```

Comment:
don't use`public` in bindings (it requires other types to be public and so on...) unless it's required (fail to compile).